src/components/CowPenContextMenu/CowPenContextMenu.js

```diff
@@ -66,107 +67,134 @@ const CowBloodline = memo(({ colorsInBloodline }) => (
-                    ? faFullHeart
-                    : faEmptyHeart,
-                  className: classNames('heart', {
-                    'is-full': isHeartFull(i, numberOfFullHearts),
-                  }),
-                }}
-              />
-            </li>
-          ))}
-        </ol>
-        <Tooltip
-          {...{
-            arrow: true,
-            placement: 'top',
-            title: `Check this box to put ${cow.name} into a ${huggingMachine.name} and automatically hug them at the start of every day. Requires a Hugging Machine in your inventory.`,
-          }}
-        >
-          <FormControlLabel
-            {...{
-              control: (
-                <Checkbox
+      <p>Weight: {getCowWeight(cow)} lbs.</p>
+      {isCowPurchased && (
+        <>
+          <ol className="hearts">
+            {nullArray(10).map((_null, i) => (
+              <li key={`${cow.id}_${i}`}>
+                <FontAwesomeIcon
                   {...{
-                    color: 'primary',
-                    // TODO: This Boolean cast is needed for backwards
-                    // compatibility. Remove it after 10/1/2020.
-                    checked: Boolean(cow.isUsingHuggingMachine),
-                    onChange: e => handleCowAutomaticHugChange(e, cow),
+                    icon: isHeartFull(i, numberOfFullHearts)
+                      ? faFullHeart
+                      : faEmptyHeart,
+                    className: classNames('heart', {
+                      'is-full': isHeartFull(i, numberOfFullHearts),
+                    }),
                   }}
                 />
-              ),
-              disabled: !cow.isUsingHuggingMachine && !huggingMachinesRemain,
-              label: 'Hug automatically',
+              </li>
+            ))}
+          </ol>
+          <Tooltip
+            {...{
+              arrow: true,
+              placement: 'top',
+              title: `Check this box to put ${cow.name} into a ${huggingMachine.name} and automatically hug them at the start of every day. Requires a Hugging Machine in your inventory.`,
             }}
-          />
-        </Tooltip>
-        <Tooltip
-          {...{
-            arrow: true,
-            placement: 'top',
-            title: isInBreedingPen
-              ? `Uncheck this box to return ${cow.name} to the regular pen.`
-              : `Check this box to move ${
-                  cow.name
-                } to the breeding pen to mate with a ${
-                  cow.gender === genders.MALE ? 'female' : 'male'
-                } cow.`,
-          }}
-        >
-          <FormControlLabel
+          >
+            <FormControlLabel
+              {...{
+                control: (
+                  <Checkbox
+                    {...{
+                      color: 'primary',
+                      // TODO: This Boolean cast is needed for backwards
+                      // compatibility. Remove it after 10/1/2020.
```

Comment:
can be removed?

Reply:
Good catch! Thanks for calling this out. Removed in https://github.com/jeremyckahn/farmhand/commit/b438824f0debafd6e12df93d93e697f069641a53!